sts-mock/template.yaml

```diff
@@ -291,4 +329,12 @@ Resources:
               ArnLike:
                 aws:SourceArn: !GetAtt JwksBucket.Arn
               StringEquals:
-                aws:SourceAccount: !Ref AWS::AccountId
\ No newline at end of file
+                aws:SourceAccount: !Ref AWS::AccountId
+
+Outputs:
+  StsMockApiUrl:
+    Description: STS Mock API Gateway DNS
+    Value: !Sub https://${StsMockApiDomainName}
+    Export:
```

Comment:
I thought it might be nice exporting this so that it can be referenced in the async backend SAM template - but I can remove it if not necessary/won't work...

Reply:
We should export it, at least so someone can easily see in Cloudformation what the DNS is. We need to be wary about importing across templates as it creates tight application couplings, despite its benefits. One to figure out as we go along

Reply:
What we will definitely want is a script that can:
1) Infer what the DNS should be based off the format we've used
2) Deploy the application stacks in parallel

Reply:
Good point 🤔 -  in that case just ignore me 😄